crates/runtime/src/init/model.rs

```diff
@@ -44,6 +44,11 @@ pub enum Error {
         "Failed to load model {name} from spicepod.\nUnable to determine model type. Verify the model source and try again.\nFor details, visit https://spiceai.org/docs/components/models",
     ))]
     UnableToDetermineModelType { name: String },
+
+    #[snafu(display(
+        "Model {name} includes a non-existent path: {path}.\nVerify the model configuration and ensure all paths are correct.\nFor more information, visit https://spiceai.org/docs/components/models",
```

Comment:
```suggestion
        "Model {name} includes a non-existent path: {path}.\nVerify the model configuration and ensure all paths are correct.\nFor details, visit https://spiceai.org/docs/components/models",
```

Consistency

Reply:
Thank you, adde the same change in parallel :)